doc/modules/neighbors.rst

```diff
@@ -427,6 +482,29 @@ and the ``'effective_metric_'`` is in the ``'VALID_METRICS'`` list of
 same order as the number of training points, and that ``leaf_size`` is 
 close to its default value of ``30``.
 
+Valid Metrics for Nearest Neighbor Algorithms
+---------------------------------------------
```

Comment:
This shows the attributes to the users, but also makes sure that a doctests fails in this place if someone modifies the valid metrics.